modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -0,0 +1,216 @@
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+.Prerequisites
+
+
+.Procedure
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+. Create a NMState resource that contains the decoded base64 information for your customized `br-ex` bridge network. 
++
+.Example of an NMState resource YAML file with a configuration for a customized `br-ex` bridge network
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+# ...
+    interfaces:
+    - name: enp2s0 <1>
+      type: ethernet <2>
+      state: up <3>
+      ipv4:
+        enabled: false <4>
+      ipv6:
+        enabled: false
+    - name: br-ex
+      type: ovs-bridge
+      state: up
+      ipv4:
+        enabled: false
+        dhcp: false
+      ipv6:
+        enabled: false
+        dhcp: false
+      bridge:
+        port:
+        - name: enp2s0 <5>
+        - name: br-ex
+      - name: br-ex
+      type: ovs-interface
+      state: up
+      copy-mac-from: enp2s0
+      ipv4:
+        enabled: true
+        dhcp: true
+      ipv6:
+        enabled: false
+        dhcp: false
+# ...
+----
+<1> Name of the interface.
+<2> The type of interface. This example creates a bridge.
+<3> The requested state for the interface after creation.
+<4> Disables IPv4 and IPv6 in this example.
+<5> The node NIC to which the bridge attaches.
+
+. Use the `cat` command to base64-encode the contents of the NMState resource YAML file:
++
+[source,terminal]
+----
+$ cat <base64_decoded_nmstate_configuration>.yaml | base64 <1>
+----
+<1> Replace `<base64_decoded_nmstate_configuration>` with the name of your NMState resource YAML file.
+
+. Create a `MachineConfig` manifest file and define a customized `br-ex` bridge network configuration analogous to the following example:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker <1>
+  name: 10-br-ex-worker <2>
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<base64_encoded_nmstate_configuration> <3>
+        mode: 0644
+        overwrite: true
+        path: /etc/nmstate/openshift/cluster.yml
+# ...
+----
+<1> For each node in your cluster, specify the hostname path to your node and the base-64 encoded Ignition configuration file data for the machine type. If you have a single global configuration specified in an `/etc/nmstate/openshift/cluster.yml` configuration file that you want to apply to all nodes in your cluster, you do not need to specify the hostname path for each node. The `worker` role is the default role for nodes in your cluster. Note that the `.yaml` extension does not work when specifying the hostname path for each node or all nodes in the `MachineConfig` manifest file.
+<2> The name of the policy.
+<3> Pulls in the encoded base64 NMState configuration into the `MachineConfig` manifest file.
```

Comment:
It just writes the decoded content to the specified path. There's a systemd service that actually applies the config after that.